src/kirby/components/empty-state/empty-state.component.tns.html

```diff
@@ -0,0 +1,10 @@
+<FlexboxLayout flexDirection="column" justifyContent="space-around">
+  <StackLayout class="box" alignSelf="center">
+    <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true" class="icon"></kirby-icon>
```

Comment:
Remove unused class:
```suggestion
    <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true"></kirby-icon>
```

Reply:
It is used again now